docs/whats-new/release-notes/etendo-classic/bundles/localization-spain-extensions/release-notes.md

```diff
@@ -13,6 +13,7 @@ tags:
 
 | Versión | Fecha de Publicación | Desde Core | Hasta Core | Estado | GitHub|
 | :--- | :--- | :--- | :--- | :---: | :---: |
+| [1.9.0](#190) | 06/02/2024 | 22.4.3 | 23.4.x | CS | :white_check_mark:|
 | [1.8.0](#180) | 26/01/2024 | 22.4.3 | 23.4.x | CS | :white_check_mark:|
```

Comment:
@RubenEtendo Is it correct that both are CS?

Reply:
No, you are right. I'll make the change